xrshim.js

```diff
@@ -0,0 +1,30 @@
+const REALAPI_URL = 'https://raw.githack.com/takahirox/webxr-ios-js/PolyfillUpdateWIP/dist/webxr.js';
```

Comment:
Loading from my working repository so far. Don't forget to edit the path if you locally edit the polyfill and want to load it.

Reply:
Did you add support for inline sessions?

Reply:
Not exactly.

The shim and polyfill return `true` from `isSessionSupported()` even for `inline` to follow the WebXR API spec

> The user-agent MUST have an inline XR Device, which is an XR Device that MUST contain "inline" in its list of supported modes. 

https://immersive-web.github.io/webxr/#xr-device-concept

but no distinguish between `inline` session and `immersive-ar` session in the polyfill for now. TBH, I'm not really sure how `inline` session should act yet (no exposing AR specific APIs?). I want to update for it later if needed and instead add comment so far.

Another option may be, although it doesn't follow the spec, returning `false` from `isSessionSupported()` for `inline`.

What do you think?

Reply:
The way inline will probably work is that we will only pass a subset of the camera/viewer tracking info (there are still discussions as to what that would mean;  might just be orientation, but could include position), but no other info.  We wouldn't display the video (although right now it doesn't matter if we do, we just won't see it) and wouldn't create the full-screen divs, etc.   

The page would use the viewer tracking info to render into a webgl canvas, without the video.

So, for now, I'd prefer to say "no" for inline, until that works.

Reply:
OK, agreed. I update the PR to refuse 'inline'.

Reply:
Updated. Thanks for the comment.